src/js/options.js

```diff
@@ -79,104 +86,17 @@ function mainConfigurationUpdated(lazy) {
 // --- Header --- //
 
 var Header = {
-  TABS: [
```

Comment:
The complete `Header` object should be removed from `options.js`:

* All the code from `Header.update` should be an `initialize` function inside the header's component, called on the `onMount` event. Also, you should use Svelte code for define the click behavior of tabs (i.e. `on:click`), instead of the vanilla `addEventListener` function).
* The code from `Header.tabClicked` should just be an internal function from the header's component (you'll do something like `on:click={tabClicked(tab)}`.

> You'll have, however, to notify `options.js` when a tab is clicked, in order to invoke `updateTabsVisibility`, but this can be done using [component events](https://svelte.dev/tutorial/component-events): on the header component you do something like `dispatch('tabClicked', tabName)` and on `options.js` you do something like `headerComponent.$on("tabClicked", updateTabsVisibility)`.

> Also note that the code that initializes the footer (inside `Header.update`) has been removed by #110